test/test_scala_kernel.py

```diff
@@ -96,11 +102,11 @@ def test_init_magic_completion(spylon_kernel):
     assert set(result['matches']) == {'launcher.conf.spark.executor.cores'}
 
 
-@pytest.mark.skip("fails randomly, maybe because interpreter is reused")
-def test_stderr(spylon_kernel):
+@pytest.mark.skip('fails randomly, possibly because of mock reuse across tests')
```

Comment:
In-person is a good idea. Let's do that tomorrow.

Summary: I spent a good couple of hours trying to track down why this test is flaky and a better way to write it. I wound up back where I started with this test disabled. Which isn't to say that stdout/err are untested: they are as part of the jupyter_kernel_test suite that spawns a kernel and puts it through the paces before any of the spylon specific tests.